requirements.txt

```diff
@@ -0,0 +1,6 @@
+ops==2.6.0
+pyyaml
+charmhelpers
+jinja2
+cryptography
+git+https://github.com/canonical/ops-lib-nrpe.git#egg=ops-lib-nrpe
```

Comment:
Can we pin a particular revision rather than taking latest?

Reply:
Will do `pip freeze` after I get 1 CI run passing

Reply:
This looks pretty old -- is there a reason to use this rather than build an integration with COS Lite instead?

Reply:
The aim was just to prevent as many breaking changes for now as possible. This should however be covered in the second iteration where we will look to replace this with COS and replace the snap/systemd operation with charmlibs as well.